examples/end-to-end-benchmarks.hs

```diff
@@ -12,14 +12,15 @@ import System.Process (system)
 
 benchmarks :: [Benchmark]
 benchmarks =
-    [ bgroup "roundrip"
+    [ bgroup "roundtrip"
         [ bench "ping" (nfIO (system "ping -c1 8.8.8.8 > /dev/null")) ]
     ]
 
 main :: IO ()
 main = defaultMainWith config "hyperion-example-end-to-end" benchmarks
   where
     config = defaultConfig
-      { configMonoidSamplingStrategy = return $ timeBound (5 * secs) (repeat 10)
+      { configMonoidSamplingStrategy =
+          pure $ timeBound (fromIntegral (5 * secs)) (repeat 10)
```

Comment:
Couldn't we have something more explicit than `fromIntegral` here, to make the units a bit more explicit in the code?